modeltranslation/manager.py

```diff
@@ -429,6 +429,7 @@ def values(self, *fields, **expressions):
         if not fields:
             # Emulate original queryset behaviour: get all fields that are not translation fields
             fields = self._get_original_fields()
+        fields += tuple(expressions)
```

Comment:
This is based on https://github.com/django/django/blob/2fd755b361d3da2cd0440fc9839feb2bb69b027b/django/db/models/query.py#L1304